internal/clientconn/conn.go

```diff
@@ -69,6 +69,7 @@ type newConnOpts struct {
 	proxyAddr       string
 	mode            Mode
 	handlersMetrics *handlers.Metrics
+	startUnixTime   int64
```

Comment:
I think time manipulation will be easier if we store original `time.Time` value

Reply:
Good point. Fixed.